en/appendices/3-4-migration-guide.rst

```diff
@@ -156,7 +156,8 @@ The following is a list of methods that are deprecated and replaced with
     * ``config()``
     * setter part of ``canBeJoined()`` (now ``setCanBeJoined()``)
 ``Cake\ORM\EagerLoader``
-    * ``matching()``
+    * ``matching()`` (``getMatching()`` will have to be call after ``setMatching()`` to
```

Comment:
I updated #4593 with this, any native English speaker can correct me if I'm wrong :)